test/irmin-graphql/common.mli

```diff
@@ -59,12 +59,16 @@ type query
     For example, the following query returns the latest commit hash for the
     [main] branch:
 
-    {[ query (func "main" (field "hash")) ]}
+    {[
```

Comment:
What's the logic behind ocamlformat inserting blank lines at the beginning and end of codeblocks?

Reply:
Thanks! So this is some extra refinement of ocamldoc syntax implemented by odoc. Is the restriction documented anywhere? (Presumably the blank lines are not considered part of the actual code?)

Reply:
I don't think it is documented anywhere, it is more of a practical choice for the formatting, so we don't have to check whether the code is big enough to break or if it was initially breaking and whether we should preserve it, we break everytime.

It looks like `odoc-parser` is trimming the leading and trailing blanklines in the code block.